src/components/userDasboard/components/DashboardDetails.jsx

```diff
@@ -9,23 +8,25 @@ const DashboardDetails = ({ numberOfOrgs, numberOfDictionary, organizations }) =
     return (
       <div className="user-data">
         <p className="lead">
-          You belong to {nameOfOrganizations} organization. This can be changed via{' '}
-          <a href="https://qa.openconceptlab.org/" target="_blank" rel="noopener noreferrer">
+          <span>
+            You belong to {nameOfOrganizations} organization. This can be changed via{' '}
+            <a href="https://qa.openconceptlab.org/" target="_blank" rel="noopener noreferrer">
             the traditional OCL
-          </a>.
-        </p>
-        <p className="lead">
```

Comment:
Isn't this a change in the UI?

Reply:
When I reverted the changes, alexmoch PR had been merged and I had a different implementation that is why it appears as if I touched the UI